packages/nextjs/components/MenuButtonComponent/MenuButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { menuButtonClasses } from "./MenuButton.styles";
+
+interface MenuButtonProps {
+  label: string;
+  onClick: () => void;
+  isActive?: boolean;
+  className?: string;
+  disabled?: boolean;
+}
+
+const MenuButton: React.FC<MenuButtonProps> = ({
```

Comment:
```suggestion
export const MenuButton: React.FC<MenuButtonProps> = ({
```

lets export this as const no need for default export

Reply:
it has been resolved.